### PR TITLE
Use the project name within the test buckets created for integration tests

### PR DESCRIPTION
**Pull request by jskeet**
This change includes batch files to clean up after the tests, and also to set up new test buckets.
(The data required is in a data subdirectory.) The scripts are crude, but they do the job to start with...

### CODE REVIEW

File: GoogleApiWrappers/src/Google.Apis.Storage.v1.IntegrationTests/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+====
+
+The integration tests require:
+
+- A project, identified by the `TEST_PROJECT` environment variable
+- An authenticated account with read/write access to the project
+- A number of storage buckets with canned data in
+
+The last bullet here can be set up using `setup.bat`, and deleted
+with `clean.bat`. Running `setup.bat` will automatically run
```
Comment (ivannaranjo) [inline GoogleApiWrappers/src/Google.Apis.Storage.v1.IntegrationTests/README.md]: General though, maybe we can write these shared scripts between windows and linux/osx as python scripts that can run xplatform? Similar to how gcloud itself is implemented?

Comment (ivannaranjo) [inline GoogleApiWrappers/src/Google.Apis.Storage.v1.IntegrationTests/README.md]: I take this back, in this case the batch files are so simple is not worth the effort.

**Comment by ivannaranjo** [comment]
Go for it :shipit:

**Comment by SurferJeffAtGoogle** [comment]
In our samples, we've been using the environment variable names GOOGLE_PROJECT_ID and GOOGLE_BUCKET.  See https://github.com/GoogleCloudPlatform/csharp-docs-samples/blob/master/storage/api/test/Test.cs

**Comment by jskeet** [comment]
Okay. Will consider whether switching to that would be a good idea. It's obviously easy to do - it's only a pain if someone wants to use a different project for the different scenarios, which I guess is unlikely.

**Pull request merged**